test/Microsoft.DotNet.ApiSymbolExtensions.Tests/Microsoft.DotNet.ApiSymbolExtensions.Tests.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)test\Microsoft.DotNet.ApiCompatibility.Tests\Microsoft.DotNet.ApiCompatibility.Tests.csproj" />
```

Comment:
The ApiSymbolExtensions library nor the tests should depend on APICompat or the ApiCompatibility library. That's the wrong layer.

Reply:
What would be a good mock logging class to use? Or maybe I could move SuppressibleTestLog to a higher level project.

Reply:
The hierarchy is:

![image](https://github.com/user-attachments/assets/deb142fe-4e6f-48e9-a085-7da7827157fe)

Reply:
```mermaid
flowchart TD

subgraph APICompat
    A[APICompat.Tool] --> C(PackageValidation)
    A --> D(ApiCompatibility)

    B[APICompat.Task] --> C
    B --> D

    C --> D
end

subgraph GenAPI
    F[GenAPI.Tool] --> H[GenAPI]
    G[GenAPI.Task] --> H[GenAPI]
end

subgraph ApiSymbolExtensions
    D --> E(ApiSymbolExtensions)
    H --> E
end

```

Reply:
Mock's cool. Only minimal code modifications needed when using that.

Reply:
```suggestion
```